jsonnet/patch-rules.libsonnet

```diff
@@ -65,6 +65,12 @@ local excludedRules = [
       { alert: 'ThanosQueryRangeLatencyHigh' },
     ],
   },
+  {
+    name: 'kubernetes-storage',
+    rules: [
+      { alert: 'KubePersistentVolumeFillingUp' },
```

Comment:
you can use `patchedRules` to patch only the PromQL expression.

Reply:
@simonpasquier , it is bit tricky here as both of the problematic alerts[1][2] having same name.

[1] https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/43c78a647d6c229ec2d0de54a82c0bc40bc8a9b2/alerts/storage_alerts.libsonnet#L20
[2] https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/43c78a647d6c229ec2d0de54a82c0bc40bc8a9b2/alerts/storage_alerts.libsonnet#L37

Reply:
@simonpasquier  IMHO, https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/43c78a647d6c229ec2d0de54a82c0bc40bc8a9b2/alerts/storage_alerts.libsonnet#L37 alert should be renamed as `KubePersistentVolumeFillingUpSoon`. WDYT?

Reply:
ah I see! The reason why both alerts have the same name is because they use different `severity` and with [inhibition rules](https://github.com/openshift/cluster-monitoring-operator/blob/5cfda856c05632f0f607bac5f831b82a76a046e4/assets/alertmanager/secret.yaml#L17-L23), the warning one will be muted if the critical alert fires (so the user gets the most accurate information).

Reply:
Ideally `patchOrExcludeRule` should be able to match on alert name + severity label but it requires more changes indeed...